cmd/notation/key.go

```diff
@@ -48,6 +48,20 @@ func keyCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "key",
 		Short: "Manage keys used for signing",
+		Long: `Manage keys used for signing
+
+Example - Add a key to signing key list:
+  notation key add --plugin <plugin_name> --id <key_id> <key_name>
+
+Example - List keys used for signing:
+  notation key ls
+
+Example - Update the default signing key:
+  notation key set --default <key_name>
+
+Example - Delete the key from signing key list:
+  notation key rm <key_name>...
```

Comment:
can we use `notation key delete`, not the alias

Reply:
The same reason explained above